.gitignore

```diff
@@ -5,6 +5,7 @@
 .idea/
 .vs/
 .DS_Store
+DevAuthToken.txt*
```

Comment:
Why the wildcard at the end? Is this stored in the root project directory?

Reply:
because of the meta file that unity creates

Reply:
:(